la_palma_overview/__init__.py

```diff
@@ -93,30 +114,32 @@ def smart_fact2img(rows, cols):
         ' Zenith....... {source_zd:.1f} {source_zd_unit:s}\n'
     ).format(
         Magnitude=sfc.sqm().magnitude.value,
-        power=sfc.sipm_currents().power.value,
-        power_unit=sfc.sipm_currents().power.unit,
-        min_cur=sfc.sipm_currents().min_per_sipm.value,
-        med_cur=sfc.sipm_currents().median_per_sipm.value,
-        max_cur=sfc.sipm_currents().max_per_sipm.value,
-        cur_unit=sfc.sipm_currents().max_per_sipm.unit,
-        out_temp=sfc.weather().temperature.value,
+        power=currents.power.value,
```

Comment:
Ah, yes. Sorry, to big of a commit. The originial reason was line length, but then I also reduced the number of smartfact calls.